### ISSUE TITLE
No aria-label or placeholder text defined for warehouse search fields

### ISSUE THREAD

**Issue by frobel** [shopware/platform#7631]
*PHP Version:* 8.3   | *Shopware Version:* 6.7  | *Affected area / extension:* Extension:Commercial 

 ----

*_Actual behaviour:_*

No aria-label or placeholder text defined for warehouse search fields.

![Image](https://github.com/user-attachments/assets/e4c25fe0-d4cd-4319-9ec3-8b17ba41b50d)

*_Expected behaviour:_*

At least aria-label or placeholder from A11Y perspective should be defined.

*_How to reproduce:_*

1. Commercial installed
2. Goto settings > warehouses
3. check the DOM of the search fields

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-41159**



[created from NEXT-41159, comment 545532]

**Comment by shopwareBot** [comment]
[public] 

 > Priority changed to: **High**

 Thank you for your report. Based on the information in this report, we have decided to investigate this issue in more detail and are planning to provide a fix for this issue. Although we are always eager to solve such problems as soon as we can, we can't give you a more specific time frame right now. We appreciate your patience and thank you for your understanding.



[created from NEXT-41159, comment 545579]
